README.md

```diff
@@ -117,3 +117,11 @@ You should see unlocked padlock icons on the right of every endpoint.
 4. Select 'Bearer Token' from the Auth Type dropdown list
 5. Paste the token into the box
 6. Postman will add the correct header
+
+## Using the vis.gl/react-google-maps API
```

Comment:
I think it would be good to have a message under this heading to say that you can run the app without the API key explaining which elements won't work if you do that.